### ISSUE TITLE
Typo in TypeScript `pm:drag` event (latLng/latlng)

### ISSUE THREAD

**Issue by lmachens** [geoman-io/leaflet-geoman#972]
Hello 👋,

the type definition for `pm:drag` event is invalid. It says `latLng`, but it is actually `latlng`.

```diff
export type DragEventHandler = (e: {
      layer: L.Layer;
      containerPoint: any;
-      latLng: L.LatLng;
+      latlng: L.LatLng;
      layerPoint: L.Point;
      originalEvent: any;
      shape: PM.SUPPORTED_SHAPES;
    }) => void;
```

You can see the difference in this fiddle:
https://jsfiddle.net/4zotduv9/5/

I think it should be `latLng`, but this requires a change in JS.

**Comment by Falke-Design** [comment]
Thank you for repoting. I created a PR and it will be fixed in the next Release

**Comment by Falke-Design** [comment]
Fixed with Release 2.11.3
